src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -1070,7 +1070,18 @@ public function insert_posts( $items = array() ) {
 				$event['post_parent'] = $k;
 			}
 
-			//if we should create a venue or use existing
+			// Do we have an existing venue for this event that we should preserve?
+			// @todo review: should we care about the potential for multiple venue IDs?
+			if (
+				! empty( $event['ID'] )
+				&& 'preserve_changes' === $update_authority_setting
+				&& $existing_venue_id = tribe_get_venue_id( $event['ID'] )
+			) {
+				$event['EventVenueID'] = $existing_venue_id;
+				unset( $event['Venue'] );
```

Comment:
This appears to be using the `retain` authority settings rather than the `preserve_changes` setting. For proper adherence to `preserve_changes`, venues and organizers need local changes tracked (similar to [here](https://github.com/moderntribe/the-events-calendar/blob/51cf2cdcf4fd30d9c1f20201382b262ce95635fd/src/Tribe/Importer/File_Importer_Events.php#L88)) and observed on update, updating only non-changed fields.